build/bin/sage-logger

```diff
@@ -63,21 +63,19 @@ fi
 
 timefile="$logdir/$logname.time"
 rm -f "$timefile"
-if /usr/bin/time -h -o /dev/null true 2>/dev/null; then
-    TIME="/usr/bin/time -h -o $timefile"
-else
-    TIME=""
-fi
+time_cmd() {
+    local max=$(($SECONDS+10))
+    exec 3>&1 4>&2
+    local out
+    TIME_OUTPUT=$((time 1>&3 2>&4 sh -c "$1") 2>&1)
+    local retstat=$?
+    [ "$SECONDS" -lt "$max" ] || echo "$TIME_OUTPUT" > "$timefile"
```

Comment:
```suggestion
    [ "$SECONDS" -lt "$max" ] || echo "$TIME_OUTPUT" >&2
```
you don't really need to go via "$timefile", do you?

It seems more natural if `time_cmd` has similar semantics/usage as regular `time` (except the time is only printed out if >= 10s).

Reply:
Yes, I do. This thing runs in a subshell.

Reply:
My point is: print to stderr instead of saving to a file. What's wrong with that? Saves a point of failure (communication via filesystem).

Reply:
From my experience as the principal maintainer of the Sage distro build system, let me just say that this concern ("point of failure") is wildly beside the point.